plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/configrepo/contract/material/CRFilter.java

```diff
@@ -9,15 +9,20 @@
 
 public class CRFilter extends CRBase {
     private List<String> ignore  = new ArrayList<String>();
-    // room for whitelist
+    private List<String> whitelist  = new ArrayList<String>();
```

Comment:
Seems odd that we're storing both and toggling on presence of one of the objects. Can't we just remember the `whitelist` boolean value instead?